management/src/test/java/org/ehcache/management/providers/settings/EhcacheSettingsProviderTest.java

```diff
@@ -106,7 +104,9 @@ public void test_standalone_ehcache() throws IOException {
         .using(new DefaultPersistenceConfiguration(ROOT.newFolder("test_standalone_ehcache")))
         .using(new DefaultManagementRegistryConfiguration()
             .addConfiguration(statisticsProviderConfiguration)
-            .setCacheManagerAlias("my-cm-1"))
+            .setCacheManagerAlias("my-cm-1")
+            .addTag("boo")
+            .addTags("foo", "baz"))
         .withCache("cache-1", cacheConfiguration1)
         .withCache("cache-2", cacheConfiguration2)
         .build(false);
```

Comment:
so I know it's not from this PR, but I suggest we change the json comparison (because the current one is limited to comparing 1 line long json document, that are not really convenient to read...)
line #121 : 

    JsonNode expectedJsonObject = mapper.readTree(expected);
    JsonNode actualJsonObject = mapper.readTree(actual);
    assertEquals(expectedJsonObject, actualJsonObject);

and then line #138 : 

      return scanner.useDelimiter("\\A").next();